aten/src/ATen/native/mps/operations/View.mm

```diff
@@ -19,10 +19,12 @@
 };
 
 static std::string getStridedKey(const ScalarType& dtype, const IntArrayRef& base_shape,
-                          const IntArrayRef& new_shape, bool is_scatter)
+                                 const IntArrayRef& new_shape, const IntArrayRef& stride,
+                                 int64_t storage_offset, bool is_scatter)
```

Comment:
Storage offset is always positive value, isn't it?
```suggestion
                                 uint64_t storage_offset, bool is_scatter)
```

Reply:
> Storage offset is always positive value, isn't it?

Yes, this is right - but this follows the way PyTorch represents the `storage_offset` (also as int64_t):
```  
int64_t storage_offset() const {
    return impl_->storage_offset();
}
```

https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/core/TensorBase.h#L118